presto-product-tests/conf/docker/files/run-tempto.sh

```diff
@@ -5,6 +5,7 @@ set -euo pipefail
 DOCKER_TEMPTO_CONF_DIR="/docker/presto-product-tests/conf/tempto"
 TEMPTO_CONFIG_FILES="tempto-configuration.yaml" # this comes from classpath
 TEMPTO_CONFIG_FILES="${TEMPTO_CONFIG_FILES},${DOCKER_TEMPTO_CONF_DIR}/tempto-configuration-for-docker-default.yaml"
+START_TIME=$(date +%Y:%m:%dT%H:%M:%S)
```

Comment:
* quotes
* `y-m-d` instead of `y:m:d` (?)

```suggestion
START_TIME=$(date "+%Y-%m-%dT%H:%M:%S")
```